pkg/webui/console/views/app/index.js

```diff
@@ -111,18 +113,33 @@ const Layout = () => {
     document.body.classList.remove(style.scrollLock)
   }, [])
 
+  const handleMouseMove = useCallback(
+    e => {
+      if (e.clientX <= 20 && isMinimized) {
+        // If the mouse is within 20px of the left edge
+        setIsDrawerOpen(true)
```

Comment:
There should be a small delay of around 150ms after which the sidebar is opened. This is to avoid sidebar flashes when the mouse is moved there briefly accidentally.

Reply:
Also the sidebar drawer should close again when:

*   It is not being hovered anymore for 300ms
*   OR immediately if the mouse travels more that 300 pixel away from the sidebar